### DISCUSSION TITLE
EDDI tab that shows a history or what EDDI said.

### DISCUSSION THREAD

**Discussion by Jessebob42** [EDCD/EDDI#2583] [Needs thought]
I occasionally can't understand or hear properly what EDDI Speech Responder Text-to-speech monitor says.  

It would be nice to be able to alt-tab to EDDI, or bring up on my display 2, to see what text was just spoken.  This could be something like the Crime Monitor history tab. 

It doesn't have to store a long history, just a page would be good enough. 

Thanks.

**Comment by Tkael** [comment]
You can use the `speechresponder.out` option on the `Speech Responder` tab to view a written record of the text that was last spoken. If enabled, the file is written to %appdata%/EDDI and can be opened by any basic text editor.
![image](https://github.com/EDCD/EDDI/assets/4382539/22643459-63c7-4ab9-ab33-91fdc0005da3)

**Comment by Jessebob42** [comment]
Sounds a bit cumbersome. I was hoping for a tab, much like the Crime Monitor (updates each new crime, yeah), to keep a running record I could glance at. As opposed to opening up a Notepad and loading the sppechresponder.out each time something new was said. 
No worries. It was just a thought. It could possibly be a good resolution to the person who wanted Closed Captions. Who knows. 
Cheers Brian!
Jessea.k.a. Commander Zund


    On Tuesday, September 19, 2023 at 02:28:45 PM GMT+1, Brian Wilson ***@***.***> wrote:  
 
 


Closed #2583 as resolved.

**Reply by Tkael** [reply]
You may not agree but I actually think SpeechResponder.out is easier and more convenient than building a speech history tab in EDDI. You can keep as much or as little history as you wish (just leave the box checked if you'd like a complete record of everything said), view the output in any text reader, and integrate the output into any closed captioning or screen overlay software that can accept a text file input.
